payara/ecosystem-vscode-plugin
# ECOSYS-108 Server log disconnects on Payara Server restart from VScode output window

# Description
This is a fix to reconnect server.log to VSCode output console on restarting the Payara Server.

# Test
- Install Node and Yarn
- Open a terminal and run `yarn install`  to install all dependencies for the project.
- Run `code .` command from the terminal to open the project in VS Code.
- Press `F5` from Code to run the project which will start another instance of Code with an installed plugin.
- Press `Ctrl + Shift + P` and search for `Add Payara Server` option.
- Right-click on the server to perform start and restart operation.
- Verify the output console window for active trailed log on restart operation.
